src/components/PurchaseSummary/style.tsx

```diff
@@ -0,0 +1,119 @@
+import styled from 'styled-components'

```

Comment:
There are some lint errors in this file.
You need to adjust the pattern of styled component names too.